### PR TITLE
Login

### PR DESCRIPTION
**Pull request by ValentinCop**

### CODE REVIEW

**Review by ghost — DISMISSED** [review]
ok for me

File: frontend/src/components/Login/index.jsx [hunk 1]
```diff
@@ -1,13 +1,76 @@
...
+  };
+
+  const hSubmit = (evt) => {
+    evt.preventDefault();
+    axios
+      .post(`${import.meta.env.VITE_BACKEND_URL}/auth/login`, formData)
+      .then(({ data }) => {
+        const { user } = data;
+        dispatch({ type: "LOGIN", payload: user });
+        toast.success("Bienvenue !", {
```
Comment (LuigiMenez) [inline frontend/src/components/Login/index.jsx:27]: The toast options, you can declare them in the toastcontainer, to avoid repeating them

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Not use redux thunks

File: frontend/src/services/reducers/index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { combineReducers, createStore } from "redux";
+import user from "./user";
+
+const mainReducer = combineReducers({ user });
```
Comment (LuigiMenez) [inline frontend/src/services/reducers/index.js:4]: the combineReducers is not useful if you only have one reducer

**Review by LuigiMenez — APPROVED** [review]
Ok for me

**Review by ghost — APPROVED** [review]
Ok for me

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**